src/Microsoft.Azure.WebJobs.Host/Bindings/BindingDataProvider.cs

```diff
@@ -126,5 +120,24 @@ public static BindingDataProvider FromTemplate(string template, bool ignoreCase
         {
             return new BindingDataProvider(template, ignoreCase);
         }
+
+        private IReadOnlyDictionary<string, object> GetBindingDataFromPropertyHelpers(object value)
+        {
+            var bindingData = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
+            foreach (var propertyHelper in _propertyHelpers)
+            {
+                try
+                {
+                    object propertyValue = propertyHelper.GetValue(value);
+                    bindingData.Add(propertyHelper.Name, propertyValue);
+                }
+                catch (Exception)
+                {
+                    // Skip if accessing the property throws
```

Comment:
Should we be logging here?

Reply:
Yes I did attempt to do that, but I didn't find an available logger. This class is instantiated via `BindingDataProvider.FromType` which is called by a couple of bindings and their static helpers with no logger. I think passing a logger all the way to this type would be invasive and may break public contracts.

I could log to console directly, but it may be better to consider passing a proper logger. Do you know of a convenient way to do that? If not, it may be a follow-up item to restructure and flow down a logger.

Reply:
I'm fine with not logging for now. For the scenario in which @joshlove-msft logged this, it's not clear to me whether every invocation would throw. If so, we wouldn't want to be logging an exception on every invocation.

Reply:
That makes sense. If we did log, I'd guess it'd be a debug level logging. But I merge this for now as it sounds like we should be OK without it.